pkg/tsdb/prometheus/json_to_dataframes_testdata/range_infinity.golden.txt

```diff
@@ -0,0 +1,22 @@
+🌟 This was machine generated.  Do not edit. 🌟
```

Comment:
By convention, these should live in the `testdata` directory instead of `json_to_dataframes_testdata`.

https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
done